library/src/main/java/com/getstream/sdk/chat/adapter/MessageListItemViewHolder.java

```diff
@@ -290,33 +290,24 @@ private void configDeliveredIndicator() {
                 || !messageListItem.isMine()
                 || message.getCreatedAt().getTime() < channelState.getLastMessage().getCreatedAt().getTime()
                 || message.getType().equals(ModelType.message_ephemeral)
-                || message.getStatus() == null
                 || isThread()
                 || isEphemeral())
             return;
 
-        switch (message.getStatus()) {
-            case SENDING:
+        switch (message.getSyncStatus()) {
+            case Sync.LOCAL_ONLY:
                 pb_deliver.setVisibility(View.VISIBLE);
                 iv_deliver.setVisibility(View.GONE);
                 break;
-            case RECEIVED:
+            case Sync.SYNCED:
                 pb_deliver.setVisibility(View.GONE);
                 iv_deliver.setVisibility(View.VISIBLE);
                 break;
-            case FAILED:
+            case Sync.IN_MEMORY:
```

Comment:
Empty cases should also have breaks. And would be great to have a comment if there is a reason why it's empty.

Reply:
> Empty cases should also have breaks. And would be great to have a comment if there is a reason why it's empty.

`Sync.IN_MEMORY` should also hide `pb_deliver` and pb_deliver like `Sync.LOCAL_FAILED` case.
So it has not breaks.